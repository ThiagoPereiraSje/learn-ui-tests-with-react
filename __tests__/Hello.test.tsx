import React from "react";
import { render } from "@testing-library/react";
import Hello from "../src/packages/components/Hello";

describe("Testing Hello Component", () => {
  test("Should be render correctly", () => {
    const { container } = render(<Hello />);

    expect(container).toMatchSnapshot();
  });

  test("Your content should be 'Hello world!'", () => {
    const { container } = render(<Hello />);
    expect(container.textContent).toBe("Hello world!");
  });
});
