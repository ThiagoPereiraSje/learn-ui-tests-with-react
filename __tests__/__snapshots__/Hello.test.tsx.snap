// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Hello Component Should be render correctly 1`] = `
<div>
  <div>
    Hello world!
  </div>
</div>
`;
